compiler/p/runtime/PPCCodeSync.inc

```diff
@@ -24,23 +24,38 @@ extern "C" void *getCodeSync();
 #if defined(TR_HOST_POWER)
 #if defined(__ibmxl__)
 #define dcbf(x)  __dcbf(x)
-#define sync()  __sync()
+#define sync() __sync()
 #define isync() __isync()
-#elif (__IBMC__ || __IBMCPP__)
```

Comment:
Both old and new XLC support these 3 as builtins, so we can check for `__IBMC__ || __IBMCPP__` instead of `__ibmxl__` and leave the inline assembly for GCC.